src/CloudFoundry.CloudController.Test.Integration/LogyardTest.cs

```diff
@@ -115,6 +115,11 @@ public void LogsTest()
                 }
             }
 
+            if (client.Info.GetV1Info().Result.AppLogEndpoint == null)
+            {
+                Assert.Inconclusive("CloudFoundry target does not have a logyard endpoint");
```

Comment:
We've had issues with inconclusive - I remember using categories to decide which tests we want to run against a certain target?